galaxyproject/galaxy
# Add support for CILogon deployments in different regions than the US

Add support for CILogon deployments in different regions than the US (Australia for example) by reading the .well-defined/openid-config instead of hard coding the urls.

This patch is minimally invasive and contains backwards compability with existing CILogon configurations.

(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
1. Setup Galaxy with OIDC to CILogon
2. Try to login see if it works

These changes have been tested with The Australian CILogon deployment, which is under the domain: https://test.cilogon.aaf.edu.au

## License
